grakn-test-tools/pom.xml

```diff
@@ -69,7 +69,12 @@
             <artifactId>embedded-redis</artifactId>
             <version>${redis-test.version}</version>
         </dependency>
-        
+        <dependency>
+            <groupId>com.github.zxl0714</groupId>
+            <artifactId>redis-mock</artifactId>
```

Comment:
I am just looking for a way of getting rid of the fake "embedded" redis we are using. I still sometimes get redis running in the background even with the shutdown hook. It happens much more rarely though.